recipes/README.md

```diff
@@ -0,0 +1,54 @@
+# 🍝 Recipes
+
+This directory contains recipes that can be used to create templates and starters.
+
+## Templates vs. Starters
+
+- **Templates** are used to create **new exercises, challenges, etc. directly in this repository** using the `npm run create` command.
+- **Starters** are used to create **new projects outside of this repository** using the `npx ghcd neuefische/web-exercises/starters/<starter> my-app -i` command.
+
+The main difference content-wise is that templates `README.md` file contains an instructions template for the exercise, while starters `README.md` file is very bare-bones. See [How?](#how) for more details.
+
+> **Warning**  
+> Never edit the files in the `starters` and `templates` directories directly. Instead, edit the recipe in this folder and then [cook it using the `npm run cook <recipe>` command](#cooking-a-recipe).
+
+## Why?
+
+Basic templates and starters only contain files. But we want some more advanced variants for Create React App and Next.js that are created via automation. The `cook` script automates the creation of all templates and starters from recipes.
```

Comment:
This is a little bit confusing to me:
- In contrast to basic templates and starters, do Create React App and Next.js templates / starters contain something else than files?
- This is why it is not so clear to me **why exactly** we are in need of such a recipe: because we want to have a single source of truth (as soon as there is a version update, etc.) so that we can simply update all templates + starters automatically?